src/Commands/ClientCreatorCommand.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Vinelab\ClientGenerator\Commands;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Vinelab\ClientGenerator\Storage\ClientStorage;
+
+class ClientCreatorCommand extends Command
+{
+    /**
+     * Command for creating a new client.
+     *
+     * @var string
+     */
+    protected $commandName = 'clients:create';
```

Comment:
Just noticed that this is required by the command, it applies for the other attributes (`$commandAppName`, etc.)